internal/test/go.mod

```diff
@@ -2,19 +2,33 @@ module github.com/openshift/windows-machine-config-bootstrapper/internal/test
 
 go 1.14
 
+replace (
+	github.com/Azure/go-autorest => github.com/Azure/go-autorest v13.3.2+incompatible // Required by OLM
+	github.com/openshift/api => github.com/openshift/api v0.0.0-20200422081840-fdd1b0c14c88 // OpenShift 4.5
+	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20200422192633-6f6c07fc2a70 // OpenShift 4.5
+	k8s.io/api => k8s.io/api v0.18.2
+	k8s.io/apimachinery => k8s.io/apimachinery v0.18.2
+	k8s.io/client-go => k8s.io/client-go v0.18.2
```

Comment:
Given that `v.19.0` has been released, shouldn't we pin to that? Otherwise we have to revisit and update later.